Mathlib/Tactic/Linter/FlexibleLinter.lean

```diff
@@ -320,7 +320,7 @@ def flexible : Std.HashSet Name :=
     `Mathlib.Tactic.FieldSimp.field,
     ``Lean.Parser.Tactic.grind,
     ``Lean.Parser.Tactic.grobner,
-    ``Lean.Parser.Tactic.cutsat,
+    ``Lean.Parser.Tactic.lia,
```

Comment:
I'm not entirely sure what's going on here so just wanted to flag that this change has been made, presumably by a regex, and I wanted to be sure it was OK.